File: test/fixtures/csharp/post_xpost.cs
using System.Net.Http;
using System.Net.Http.Headers;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Post, "http://localhost:28139/api/xxxxxxxxxxxxxxxx");
request.Content = new StringContent("{\"keywords\":\"php\",\"page\":1,\"searchMode\":1}");
request.Content.Headers.ContentType = new MediaTypeHeaderValue("application/x-www-form-urlencoded");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();

File: test/fixtures/kotlin/post_xpost.kt
import java.io.IOException
import okhttp3.MediaType.Companion.toMediaType
import okhttp3.OkHttpClient
import okhttp3.Request
import okhttp3.RequestBody.Companion.toRequestBody

val client = OkHttpClient()

val MEDIA_TYPE = "application/x-www-form-urlencoded".toMediaType()

val requestBody = "{\"keywords\":\"php\",\"page\":1,\"searchMode\":1}"

val request = Request.Builder()
  .url("http://localhost:28139/api/xxxxxxxxxxxxxxxx")
  .post(requestBody.toRequestBody(MEDIA_TYPE))
  .header("Content-Type", "application/x-www-form-urlencoded")
  .build()

client.newCall(request).execute().use { response ->
  if (!response.isSuccessful) throw IOException("Unexpected code $response")
  response.body!!.string()
}
